src/rules/enforce-props-argument-name.ts

```
@@ -128,142 +176,122 @@ export const enforcePropsArgumentName = createRule<Options, MessageIds>({
+          return;
+        }
+
+        // Only check identifier parameters with type annotations
+        if (
+          param.type === AST_NODE_TYPES.Identifier &&
+          param.typeAnnotation &&
+          param.typeAnnotation.typeAnnotation
+        ) {
+          const typeName = getTypeName(param.typeAnnotation.typeAnnotation);
+
+          if (typeName && endsWithProps(typeName) && !BUILT_IN_TYPES.has(typeName)) {
```

ESLint

[error] Replace `typeName·&&·endsWithProps(typeName)·&&·!BUILT_IN_TYPES.has(typeName)` with `⏎············typeName·&&⏎············endsWithProps(typeName)·&&⏎············!BUILT_IN_TYPES.has(typeName)⏎··········`

(prettier/prettier)